templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -1173,20 +1211,20 @@ Resources:
         ProductStackName: !Sub "${AWS::StackName}"
         ProductFamilyName: "bitbucket"
         AsgToMonitor: !Ref ClusterNodeGroup
-        DatabaseIdentifier: !Ref DB
+        DatabaseIdentifier: !If [ DBEngineAurora, !Ref DBCluster, !Ref DB ]
```

Comment:
I believe for this to work with the Aurora database the reference will have to be
`!Sub ["${DBStackNamePrefix}-dbcluster", {DBStackNamePrefix: !Ref 'AWS::StackName'}]`

Reply:
I will need to test this and verify

Reply:
I tried to spin the stack up from this template and it came up successfully. What I didn't test was connecting to the service since I was behind the VPN yesterday. Would it have come up if this was incorrect?

Reply:
I've just tested it. The deploy worked but the dashboard has no data in it. I suspect that our other templates are doing this incorrectly for Aurora as well because the underlying instance ID is not exposed by the nested template. @theghostwhoforks any insights here?

<img width="2745" alt="image" src="https://user-images.githubusercontent.com/21027663/66531173-a2a5cb00-eb56-11e9-9275-4f2c36c8df99.png">

Reply:
I agree. I'll create a ticket